charts/hybridnet/values.yaml

```diff
@@ -0,0 +1,35 @@
+images:
+  hybridnet:
+    image: hybridnetdev/hybridnet
+    tag: v0.4.1
+  registryURL: "docker.io"
+
+# -- It's always encouraged to use an overlay network to finish some general jobs in hybridnet. This
+# chart will always create an "init" overlay Network/Subnet to initialize the K8s cluster.
+initNetworkConfig:
+  # -- CIDR of "init" Subnet.
+  cidr: 100.64.0.0/16
```

Comment:
why not use a private subnet?

Reply:
Because it's not friendly. Normally, if hybridnet is installed in a fully free-downloaded K8s cluster, pod will not have a subnet-specified annotation. If the init Subnet is private, extra operations need to be finished before installing hybridnet, which brings some wasted time.